sqlite_utils/db.py

```diff
@@ -1709,9 +1763,9 @@ def duplicate(self, new_name: str) -> "Table":
         if not self.exists():
             raise NoTable(f"Table {self.name} does not exist")
         with self.db.conn:
-            sql = "CREATE TABLE [{new_table}] AS SELECT * FROM [{table}];".format(
```

Comment:
lots of these refactors to just try to generally allow the escaping to work as intended while still allowing `table.name` to include a single period. Which, incidentally, I am not validating, and probably I should be validating "at most one ." as well as "if there's a dot, the database name matches something that's actually in the current connection" - although `sqlite-utils`'s lazy-ish approach to verifying state when creating Table objects makes me a little uncertain where we'd want to apply that validation.

Reply:
my comment about validation is no longer relevant, since I'm storing the schema name explicitly.